views/AppView.js

```diff
@@ -0,0 +1,70 @@
+
+var APIkey = 'AIzaSyCRxZDF8wrnm67fIS3QkhUXhJMuIWOUMco';
+
+var AppView = Backbone.View.extend({
+  el: $('body'),
+
+  events: {
+    'keyup #search': 'fetchSearch',
+    'click .search-button' : 'fetchSearch'
+  },
+
+  initialize: function () {
+    this.$nextVideos = this.$('.next-videos');
+    this.$videoPlayer = this.$('.video-player');
+    this.listenTo(this.model.get('videos'), 'add', this.renderNextVideos);
+    this.listenTo(this.model, 'change:current_video', this.renderCurrentVideo);
+    this.makeVideoList('cats');
+  },
+
+  fetchSearch: function (e) {
+    if (e.keyCode === 13) {
+      query = $("#search").val()
+      this.$nextVideos.html('');
+      this.makeVideoList(query);
+    }
+  },
+
+  makeVideoList: function(query) {
+
+//Figure out how to make this work with backbone fetch and parse instead of jQuery ajax call
+
+    appModel.set('search', query);
+
+    var url ='https://www.googleapis.com/youtube/v3/search?part=snippet&maxResults=25&q=' + query + '&type=video&key=' + APIkey;
```

Comment:
Here's where you go off the rails... Views deal with showing results, not getting them.  This is important to understand that this code should have been in the model rather than the view.